src/vmm/src/vstate/vm/x86_64.rs

```diff
@@ -134,6 +142,11 @@ impl Vm {
             ioapic,
         })
     }
+
+    /// Gets the list of MSRs to save when creating snapshots
+    pub fn msrs_to_save(&self) -> &[u32] {
+        self.arch.msrs_to_save.as_slice()
+    }
```

Comment:
Once `arch` files will be gone, this one can be gone as well. Don't see much value here.

Reply:
This one contains the `.as_slice()` call, so it is slightly different than just accessing the field. I'd prefer to keep it tbh